editoast/justfile

```
@@ -0,0 +1,29 @@
+#!/usr/bin/env just --justfile
+
+# Display available recipes
+help:
+    @just --list --unsorted
+
+# Run the application; does not start other services
+run:
+    cargo run
```

Adding `EDITOAST_CORE_SINGLE_WORKER` as core is running on all infra (hard to provide the envvar for a one-infra dedicated core worker), and previous scripts:
```suggestion
    ./assets/sprites/generate-atlas.sh
    ./assets/fonts/generate-glyphs.sh
    diesel migration run --locked-schema
    @echo "Starting editoast in single worker mode. Configure the other services accordingly"
    @echo "If you use osrd-compose add the `sw` flag"
    EDITOAST_CORE_SINGLE_WORKER=true cargo run -- runserver
```